hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlan.java

```diff
@@ -552,7 +552,8 @@ private void createIfAbsentReceiverTasklet(
                            ReceiverTasklet receiverTasklet = new ReceiverTasklet(
                                    collector, edge.getConfig().getReceiveWindowMultiplier(),
                                    getConfig().getInstanceConfig().getFlowControlPeriodMs(),
-                                   nodeEngine.getLoggingService());
+                                   nodeEngine.getLoggingService(),
+                                   "receiverFor:" + edge.destVertex().name() + "#" + edge.destOrdinal());
```

Comment:
do we need the `receiverFor`? logger name is already `ReceiverTasklet`

Reply:
We normally print only the last part of the logger name. And we do something similar in CIES.